### ISSUE TITLE
Foresty Tools: Adjust Compass

### ISSUE THREAD

**Issue by pauldpickell** [ubc-minetest-classroom/Minetest_Classroom#131]
**Describe the bug**
Open compass tool by left-clicking twice. Right click once to show the formspec to adjust the magnetic declination from 0 to 18. Click "Adjust Compass" button and Minetest exits on an error:

```
2022-07-19 19:47:38: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod 'forestry_tools' in callback on_playerReceiveFields(): C:\minetest-5.5.1-win64\bin\..\builtin\common\after.lua:31: Invalid minetest.after invocation
2022-07-19 19:47:38: ERROR[Main]: stack traceback:
2022-07-19 19:47:38: ERROR[Main]: 	[C]: in function 'assert'
2022-07-19 19:47:38: ERROR[Main]: 	C:\minetest-5.5.1-win64\bin\..\builtin\common\after.lua:31: in function 'after'
2022-07-19 19:47:38: ERROR[Main]: 	...games\Minetest_Classroom\mods\forestry-tools/compass.lua:102: in function <...games\Minetest_Classroom\mods\forestry-tools/compass.lua:72>
2022-07-19 19:47:38: ERROR[Main]: 	C:\minetest-5.5.1-win64\bin\..\builtin\game\register.lua:429: in function <C:\minetest-5.5.1-win64\bin\..\builtin\game\register.lua:415>
```

**To Reproduce**
1. Double left-click
2. Right-click
3. Change magnetic declination to 18
4. Click "Adjust Compass" button
5. See error
6. Restarting server after crash shows that the magnetic declination was set to 18.

**Expected behavior**
The magnetic declination should save and the formspec should exit with the compass bezel updated.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Minetest 5.5.1

**Comment by Lowie375** [comment]
Just took a look at the compass: this seems to be caused by a typo on line 102. Will make a quick patch

**Comment by pauldpickell** [comment]
Fixed by #132
